glossary.po

```diff
@@ -409,6 +414,12 @@ msgid ""
 "func:`Py_NewRef` function can be used to create a new :term:`strong "
 "reference`."
 msgstr ""
+"Il est recommandé d'appeler :c:func:`Py_INCREF` sur la :term:`référence "
+"empruntée`, ce qui la transforme en place en une :term:`référence forte`. "
```

Comment:
```suggestion
"empruntée`, ce qui la transforme, sans autre modification, en une :term:`référence forte`. "
```
?

Reply:
Qu'est-ce que tu veux dire ? Je ne comprends pas de quelles modifications il pourrait s'agir.

Reply:
Je ne comprends pas trop « en place » ici. Le *Larousse* écrit : 
«  En place, qui occupe une fonction qui lui donne une certaine considération ; se dit de groupes, d'institutions qui sont établis ; se dit d'un minerai non encore exploité, d'une roche in situ. »
Peut-être *in situ* alors ?